react-native-gesture-handler.d.ts

```diff
@@ -215,6 +215,7 @@ declare module 'react-native-gesture-handler' {
     enabled?: boolean;
     waitFor?: React.Ref<any> | React.Ref<any>[];
     simultaneousHandlers?: React.Ref<any> | React.Ref<any>[];
+    cancelsTouchesInView?: React.Ref<any> | React.Ref<any>[];
```

Comment:
Should be a bool?
```suggestion
    cancelsTouchesInView?: boolean;
```

Reply:
That seems to be the case - @osdnk do you mind making this small change to allow this PR to move forward?

Thanks!